src/pages/tools/MonsterSlayerPage.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import {
+    TypedUseSelectorHook,
+    useDispatch,
+    useSelector as useReduxSelector
+} from 'react-redux';
+import {
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow
+  } from '@material-ui/core';
+
+import { loadStageScores } from '../../store/stageScoreActions';
+import { StageScore } from '../../domain/stageScore';
+import { AppState } from '../../store/rootReducer';
+
+export const useSelector: TypedUseSelectorHook<AppState> = useReduxSelector;
+
+export const MonsterSlayerPage: React.FC<{}> = () => {
+    const stageScores: StageScore[] = useSelector(
+        state => state.stageScoreState.stageScores
+    );
+
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(loadStageScores());
+    }, [dispatch]);
+
+    return (
+        <Paper>
+            <h1>Monster Slayer</h1>
+            { stageScores.length ? 
```

Comment:
With just using stageScores.length javascript will only confirm that the length property is not null. In the case of no data we would get this field being a 0 records and it would try and render the table instead of the no active monster slayer found message. You will want to update the check to use the same length != 0 that you previously had